### ISSUE TITLE
CAMPD Related Resources- 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2350]
## Context 

**Required Fix:** 

1. Finished testing 2257. I'm not sure if there's anything I can fail as far as the exit disclaimer, but I don't think screen reader users will understand the purpose. They definitely will not know the link will go to an outside site if navigating by links since the span is not part of the anchor. An aria labelled by attribute might help but I would consider this an enhancement. 

**Optional Fix- Do not implement this as apart of this ticket** 
2. Also, the link opens in a new tab, accessibility best practices say to give warning https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G201, but again this is an enhancement, not required.

**Comment by aprematta** [comment]
Testing complete, aria label includes the fact it goes outside the government domain
